engine/source/materials/fail/johnson_cook/fail_johnson_b.F

```diff
@@ -0,0 +1,107 @@
+Copyright>        GNU Affero General Public License for more details.
+Copyright>
+Copyright>        You should have received a copy of the GNU Affero General Public License
+Copyright>        along with this program.  If not, see <https://www.gnu.org/licenses/>.
+Copyright>
+Copyright>
+Copyright>        Commercial Alternative: Altair Radioss Software
+Copyright>
+Copyright>        As an alternative to this open-source version, Altair also offers Altair Radioss
+Copyright>        software under a commercial license.  Contact Altair to discuss further if the
+Copyright>        commercial version may interest you: https://www.altair.com/radioss/.
+Chd|====================================================================
+Chd|  FAIL_JOHNSON_B
+Chd|-- called by -----------
+Chd|        FAIL_BEAM3
+Chd|-- calls ---------------
+Chd|====================================================================
+      SUBROUTINE FAIL_JOHNSON_B(
+     .           NEL      ,NGL      ,NUPARAM  ,UPARAM   ,
+     .           TIME     ,TSTAR    ,SVM      ,PRESSURE ,
+     .           DPLA     ,EPSD     ,OFF      ,DFMAX    ,
+     .           TDEL     ,IOUT     ,ISTDO    )
+C-----------------------------------------------
+c    Johnson-Cook failure model for standard beams (TYPE 3)
+C-----------------------------------------------
+C   I m p l i c i t   T y p e s
+C-----------------------------------------------
+#include  "implicit_f.inc"
+C-----------------------------------------------
+C   G l o b a l   P a r a m e t e r s
+C-----------------------------------------------
+#include  "comlock.inc"
+C-----------------------------------------------
+C   D u m m y   A r g u m e n t s
+C-----------------------------------------------
+      INTEGER                     ,INTENT(IN)    :: NEL     ! size of element group
+      INTEGER                     ,INTENT(IN)    :: NUPARAM ! size of parameter array
+      INTEGER                     ,INTENT(IN)    :: IOUT    ! output file unit
+      INTEGER                     ,INTENT(IN)    :: ISTDO   ! output file unit
+      INTEGER ,DIMENSION(NEL)     ,INTENT(IN)    :: NGL     ! table of element identifiers
+      my_real                     ,INTENT(IN)    :: TIME    ! current time
+      my_real ,DIMENSION(NUPARAM) ,INTENT(IN)    :: UPARAM  ! failure model parameter array
+      my_real ,DIMENSION(NEL)     ,INTENT(IN)    :: DPLA    ! plastic strain increment
+      my_real ,DIMENSION(NEL)     ,INTENT(IN)    :: SVM     ! Von Mises stress
+      my_real ,DIMENSION(NEL)     ,INTENT(IN)    :: PRESSURE! element pressure
+      my_real ,DIMENSION(NEL)     ,INTENT(IN)    :: EPSD    ! strain rate
+      my_real ,DIMENSION(NEL)     ,INTENT(IN)    :: TSTAR   ! normalized temperature
+      my_real ,DIMENSION(NEL)     ,INTENT(INOUT) :: OFF     ! element desactivation flag
+      my_real ,DIMENSION(NEL)     ,INTENT(INOUT) :: DFMAX   ! maximum damage
+      my_real ,DIMENSION(NEL)     ,INTENT(OUT)   :: TDEL    ! desactivation time
```

Comment:
Hello @MaciekWronski, 

Could you double-check the intent of TDEL? 
TDEL(i) is initialized only for some i. With INTENT(OUT), the other values may be replaced by garbage even they were initialized elsewhere.